### ISSUE TITLE
"Compare" button for achievements and colon

### ISSUE THREAD

**Issue by MakoSipper** [tfedor/AugmentedSteam#230]
Right now, the colon shows on the line at the bottom, which is a bit weird:

![image](https://user-images.githubusercontent.com/16372995/56086951-cacaa080-5e37-11e9-868b-f0f7d9979bbb.png)

```
<span>
<a href="https://steamcommunity.com/id/MakoSipper" data-miniprofile="177110720">MakoSipper</a> achieved <a href="https://store.steampowered.com/app/382900" class="es_highlight_checked es_highlighted es_highlighted_owned es_achievements">FINAL FANTASY VI<br>
<a href="https://steamcommunity.com/id/MakoSipper/stats/382900/compare/#es-compare" class="es_achievement_compare">Compare</a>
</a>:				</span>
```

If you re-position the colon and the </a> that closes the game name, it seems to fix this issue:

```
<span>
					<a href="https://steamcommunity.com/id/MakoSipper" data-miniprofile="177110720">MakoSipper</a> achieved <a href="https://store.steampowered.com/app/382900" class="es_highlight_checked es_highlighted es_highlighted_owned es_achievements">FINAL FANTASY VI
</a>:				<br>
<a href="https://steamcommunity.com/id/MakoSipper/stats/382900/compare/#es-compare" class="es_achievement_compare">Compare</a></span>
```

![image](https://user-images.githubusercontent.com/16372995/56086943-9525b780-5e37-11e9-8d8f-60ae40dc750e.png)

**Comment by MxtOUT** [comment]
Fixed with #225

**Comment by KarlCastle** [comment]
This should have been fixed by 1f7c31e20 (in #217). And the Compare link is in the correct place for me, so I think this can be closed.
